rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -254,15 +254,35 @@ namespace rocal{
     }
 
 
-    py::object wrapper_BB_cord_copy(RocalContext context, py::array_t<float> array)
+    py::object wrapper_BB_cord_copy(RocalContext context, py::array_t<double> array)
     {
         auto buf = array.request();
-        float* ptr = (float*) buf.ptr;
+        double* ptr = (double*) buf.ptr;
         // call pure C++ function
         rocalGetBoundingBoxCords(context,ptr);
         return py::cast<py::none>(Py_None);
     }
 
+    py::object wrapper_Mask_count(RocalContext context, py::array_t<int> array)
+    {
+        auto buf = array.request();
+        int* ptr = (int*) buf.ptr;
+        // call pure C++ function
+        int count = rocalGetMaskCount(context,ptr);
+        return py::cast(count);
+    }
+
+    py::object wrapper_Mask_Coordinates(RocalContext context, py::array_t<int> array_count, py::array_t<float> array)
```

Comment:
Make M & C in small letter to maintain uniformity

Reply:
Resolved in the latest commit